.github/CONTRIBUTING.md

```
@@ -1,149 +1,123 @@
+# How to Contribute?
 
-### Before You Start
+## Testing
 
-- Keep each pull request focused on a single feature or fix.
-- Avoid unnecessary formatting changes or large unrelated edits.
-- Document what changed and why in your PR description.
+* Test the latest changes to WinUtil by running the pre-release and reporting issues you are encountering to help us continually improve WinUtil!
 
----
+#### **Run the latest pre-release**
```

markdownlint-cli2 (0.23.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)